src/Evolution/Systems/Ccz4/Tags.hpp

```diff
@@ -111,6 +111,22 @@ template <size_t Dim, typename Frame, typename DataType>
 struct FieldP : db::SimpleTag {
   using type = tnsr::i<DataType, Dim, Frame>;
 };
+
+/*!
+ * \brief The conformal spatial christoffel symbols of the second kind
+ *
+ * \details We define:
+ * \f{align}
+ *     \Gamma^k_{ij} &= \bar{\gamma}^{ij} (D_{ijl} + D_{jil} - D_{lij})
```

Comment:
`\tilde{\Gamma}` and `bar` -> `tilde`

LHS `\gamma^{ij}` should be `\gamma^{kl}`